backend/services/implementations/testService.ts

```diff
@@ -81,6 +81,15 @@ class TestService implements ITestService {
         throw new Error(`Test ID ${id} not found`);
       }
       if (testToDelete.status === AssessmentStatus.DRAFT) {
+        try {
```

Comment:
hmm can we reorder the deletion (so delete test first, then images)? not a big deal but i think having images hanging in GCP is better than having tests with invalid images

Reply:
oops sorry forgot to press comment lmao

Reply:
Done!